modules/ROOT/pages/customsidebar.adoc

```diff
@@ -78,6 +78,9 @@ tinymce.init({
       icon: 'comment',
       onSetup: (api) => {
         console.log('Render panel', api.element());
+			return () => {
+		 	api.element().focus();
+			};
       },
```

Comment:
```suggestion
      onSetup: (api) => {
        console.log('Render panel', api.element());
	api.element().focus();
	return () => {
	  editor.focus();
	};
      },
```
The element would not be able to be focused when it is de-rendered, so something like this makes more sense. And focus on the element would make more sense as it is rendered.

As per the `Custom Group Toolbar` documentation:
`onSetup is a complex property. It takes a function that is passed the component’s API and should return a callback that is passed the component’s API and returns nothing. This occurs because onSetup runs whenever the component is rendered, and the returned callback is executed when the component is destroyed. This is essentially an onTeardown handler, and can be used to unbind events and callbacks.`